wallet-test/src/test/scala/org/bitcoins/wallet/dlc/DLCExecutionTest.scala

```diff
@@ -0,0 +1,214 @@
+
+        val offer = offerOpt.get
+        val accept = acceptOpt.get
+
+        val comparableInputsA = inputsA.sortBy(_.outPoint.hex)
+        val comparableInputsB =
+          inputsB
+            .sortBy(_.outPoint.hex)
+            .map(_.copy(sigs = Vector.empty)) // initiator will not have funding sigs
+
+        assert(comparableInputsA == comparableInputsB)
+
+        val fundingTxOutpoints = fundingTx.inputs.map(_.previousOutput)
+        val outpointsA = inputsA.map(_.outPoint)
+        val outpointsB = inputsB.map(_.outPoint)
+
+        assert(fundingTxOutpoints.diff(outpointsA ++ outpointsB).isEmpty)
+
+        assert(fundingTx.outputs.size == 3)
+        assert(
+          fundingTx.outputs.exists(
+            _.scriptPubKey == offer.changeAddress.scriptPubKey))
+        assert(
+          fundingTx.outputs.exists(
+            _.scriptPubKey == accept.changeAddress.scriptPubKey))
+        assert(ScriptInterpreter.checkTransaction(fundingTx))
+      }
+  }
+
+  it must "do a dlc mutual close where the initiator wins" in {
+    case (dlcWalletA: InitializedDLCWallet, dlcWalletB: InitializedDLCWallet) =>
+      val dlcA = dlcWalletA.wallet
+      val dlcB = dlcWalletB.wallet
+
+      for {
+        offerOpt <- dlcA.dlcOfferDAO.findByEventId(eventId)
+
+        closeSig <- dlcA.initDLCMutualClose(eventId, winSig)
+        tx <- dlcB.acceptDLCMutualClose(closeSig)
+      } yield {
+        assert(offerOpt.isDefined)
+        val offer = offerOpt.get
+
+        assert(tx.inputs.size == 1)
+        assert(tx.outputs.size == 1)
+        assert(tx.outputs.head.scriptPubKey == offer.finalAddress.scriptPubKey)
+        assert(ScriptInterpreter.checkTransaction(tx))
+      }
+  }
+
+  it must "do a dlc mutual close where the recipient wins" in {
+    case (dlcWalletA: InitializedDLCWallet, dlcWalletB: InitializedDLCWallet) =>
+      val dlcA = dlcWalletA.wallet
+      val dlcB = dlcWalletB.wallet
+
+      for {
+        acceptOpt <- dlcB.dlcAcceptDAO.findByEventId(eventId)
+
+        closeSig <- dlcB.initDLCMutualClose(eventId, loseSig)
+        tx <- dlcA.acceptDLCMutualClose(closeSig)
+      } yield {
+        assert(acceptOpt.isDefined)
+        val accept = acceptOpt.get
+
+        assert(tx.inputs.size == 1)
+        assert(tx.outputs.size == 1)
+        assert(tx.outputs.head.scriptPubKey == accept.finalAddress.scriptPubKey)
+        assert(ScriptInterpreter.checkTransaction(tx))
+      }
+  }
+
+  it must "do a unilateral close as the initiator" in {
+    case (dlcWalletA: InitializedDLCWallet, _: InitializedDLCWallet) =>
+      val dlcA = dlcWalletA.wallet
+
+      for {
+        (cet, closeTxOpt) <- dlcA.executeDLCForceClose(eventId, winSig)
+
+      } yield {
+        assert(closeTxOpt.isDefined)
+        val closeTx = closeTxOpt.get
+
+        assert(cet.inputs.size == 1)
+        assert(cet.outputs.size == 1)
+        assert(ScriptInterpreter.checkTransaction(cet))
+        assert(ScriptInterpreter.checkTransaction(closeTx))
```

Comment:
Is there some reason that here (and with every call to `checkTransaction` we can't use `ScriptInterpreter.runVerify` which would actually verify sigs and such? I know that this requires more than just the tx but I imagine that we would be able to get the extra info in at least some of these cases?